### ISSUE TITLE
Unable to load fabric8-console after install on Kubernetes in GKE (version 0.4.82)

### ISSUE THREAD

**Issue by gregdaybreak** [fabric8io/fabric8#6452]
I was directed to open an Issue from the IRC channel. 

I do the get started guide found here:
http://fabric8.io/guide/getStarted/gke.html

Once everything is set up I can show everything looks fine:

lc-gregbrown:Downloads gregbrown$ gofabric8 version
gofabric8, version 0.4.82 (branch: 'master', revision: '49da7d1')
  build date:       '20161014-12:39:38'
  go version:       '1.7.1'

lc-gregbrown:Downloads gregbrown$ kubectl get pods
NAME                                       READY     STATUS    RESTARTS   AGE
exposecontroller-430876756-3ow36           1/1       Running   0          21m
fabric8-docker-registry-1958215574-p4p95   1/1       Running   0          21m
fabric8-forge-1261655934-15m3y             1/1       Running   0          21m
fabric8-lzknj                              2/2       Running   0          21m
gogs-1981025962-k9g7z                      1/1       Running   0          21m
jenkins-608193230-65kj7                    1/1       Running   0          21m
nexus-3406241676-03gki                     1/1       Running   0          21m

lc-gregbrown:Downloads gregbrown$ gofabric8 validate

Validating your Kubernetes installation at https://{IP} in namespace default

Service account...............................................................✔ 
Console.......................................................................✔ 
Jenkinshift Service...........................................................✔ 
PersistentVolumeClaims........................................................✔ 
ConfigMaps....................................................................✔ 

When I go to https://{IP}/api/v1/proxy/namespaces/default/services/fabric8/workspaces

the console shows up but only shows the attached image. I see the Fabric8 logo and Teams and the main window just has a spinning wheel.

Logs are below:

[KubernetesAPI] master_url set to 'k8s', assuming proxy is being used
libs.js:120934 [KubernetesAPI] Using kubernetes API URL:  https://{IP}/k8s
https://{IP}/favicon.ico Failed to load resource: the server responded with a status of 404 ()
libs.js:120934 [hawtio-oauth] All oauth plugins have executed
https://{IP}/k8s/oapi/v1 Failed to load resource: the server responded with a status of 404 ()
libs.js:120934 [KubernetesAPI] Error probing https://{IP}/k8s/oapi/v1 assuming backend is not an openshift instance.  Error details: status:  error  errorThrown:    jqXHR instance:  Object
libs.js:120934 [kubernetes-watcher] Backend is a vanilla Kubernetes instance, namespace kind:  namespaces
https://{IP}/k8s/api/v1/namespaces Failed to load resource: the server responded with a status of 404 ()
libs.js:120934 [KubernetesAPI.WSHandler] Failed to fetch data while connecting to backend for type:  namespaces  error:  Object
libs.js:207470 WebSocket connection to 'wss://{IP}/k8s/api/v1/namespaces?watch=true&access_token' failed: Error during WebSocket handshake: Unexpected response code: 404i @ libs.js:207470
4https://{IP}/k8s/api/v1/namespaces Failed to load resource: the server responded with a status of 404 ()
libs.js:120934 [kubernetes-watcher] Error fetching namespaces:  Object
libs.js:120934 [fabric8-console] Using branding:  fabric8-branding
libs.js:120934 [fabric8-console] started, version:  1.0.73
libs.js:120934 [fabric8-console] commit ID:  0a64a2765d1961bd57e934de74c47cd6aef7563b
https://{IP}/k8s/api/v1/namespaces/default/events Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/api/v1/namespaces/default/configmaps Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/api/v1/namespaces/default/replicationcontrollers Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/api/v1/namespaces/default/pods Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/api/v1/namespaces/default/secrets Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/apis/extensions/v1beta1/ingresses Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/apis/extensions/v1beta1/namespaces/default/replicasets Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/api/v1/namespaces/default/services Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/apis/extensions/v1beta1/namespaces/default/deployments Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/api/v1/configmaps Failed to load resource: the server responded with a status of 404 ()
libs.js:120934 [KubernetesAPI.WSHandler] Failed to fetch data while connecting to backend for type:  events  error:  Object
libs.js:120934 [KubernetesAPI.WSHandler] Failed to fetch data while connecting to backend for type:  configmaps  error:  Object
libs.js:120934 [KubernetesAPI.WSHandler] Failed to fetch data while connecting to backend for type:  replicationcontrollers  error:  Object
libs.js:120934[KubernetesAPI.WSHandler] Failed to fetch data while connecting to backend for type:  pods  error:  Object
libs.js:120934[KubernetesAPI.WSHandler] Failed to fetch data while connecting to backend for type:  secrets  error:  Object
libs.js:120934[KubernetesAPI.WSHandler] Failed to fetch data while connecting to backend for type:  ingresses  error:  Object
libs.js:120934[KubernetesAPI.WSHandler] Failed to fetch data while connecting to backend for type:  replicasets  error:  Object
libs.js:120934[KubernetesAPI.WSHandler] Failed to fetch data while connecting to backend for type:  services  error:  Object
libs.js:120934[KubernetesAPI.WSHandler] Failed to fetch data while connecting to backend for type:  deployments  error:  Object
libs.js:120934[KubernetesAPI.WSHandler] Failed to fetch data while connecting to backend for type:  configmaps  error:  Object
libs.js:207470WebSocket connection to 'wss://{IP}/k8s/api/v1/namespaces/default/events?watch=true&access_token' failed: Error during WebSocket handshake: Unexpected response code: 404i @ libs.js:207470
libs.js:207470WebSocket connection to 'wss://{IP}/k8s/api/v1/namespaces/default/configmaps?watch=true&access_token' failed: Error during WebSocket handshake: Unexpected response code: 404i @ libs.js:207470
libs.js:207470WebSocket connection to 'wss://{IP}/k8s/api/v1/namespaces/default/replicationcontrollers?watch=true&access_token' failed: Error during WebSocket handshake: Unexpected response code: 404i @ libs.js:207470
https://{IP}/k8s/api/v1/namespaces/default/events Failed to load resource: the server responded with a status of 404 ()
libs.js:207470WebSocket connection to 'wss://{IP}/k8s/api/v1/namespaces/default/pods?watch=true&access_token' failed: Error during WebSocket handshake: Unexpected response code: 404i @ libs.js:207470
https://{IP}/k8s/api/v1/namespaces/default/configmaps Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/api/v1/namespaces/default/replicationcontrollers Failed to load resource: the server responded with a status of 404 ()
libs.js:207470WebSocket connection to 'wss://{IP}/k8s/api/v1/namespaces/default/secrets?watch=true&access_token' failed: Error during WebSocket handshake: Unexpected response code: 404i @ libs.js:207470
libs.js:207470WebSocket connection to 'wss://{IP}/k8s/apis/extensions/v1beta1/ingresses?watch=true&access_token' failed: Error during WebSocket handshake: Unexpected response code: 404i @ libs.js:207470
https://{IP}/k8s/api/v1/namespaces/default/pods Failed to load resource: the server responded with a status of 404 ()
libs.js:207470WebSocket connection to 'wss://{IP}/k8s/apis/extensions/v1beta1/namespaces/default/replicasets?watch=true&access_token' failed: Error during WebSocket handshake: Unexpected response code: 404i @ libs.js:207470
https://{IP}/k8s/api/v1/namespaces/default/secrets Failed to load resource: the server responded with a status of 404 ()
libs.js:207470WebSocket connection to 'wss://{IP}/k8s/api/v1/namespaces/default/services?watch=true&access_token' failed: Error during WebSocket handshake: Unexpected response code: 404i @ libs.js:207470
https://{IP}/k8s/apis/extensions/v1beta1/ingresses Failed to load resource: the server responded with a status of 404 ()
libs.js:207470WebSocket connection to 'wss://{IP}/k8s/apis/extensions/v1beta1/namespaces/default/deployments?watch=true&access_token' failed: Error during WebSocket handshake: Unexpected response code: 404i @ libs.js:207470
libs.js:207470WebSocket connection to 'wss://{IP}/k8s/api/v1/configmaps?watch=true&access_token' failed: Error during WebSocket handshake: Unexpected response code: 404i @ libs.js:207470
https://{IP}/k8s/api/v1/namespaces/default/services Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/apis/extensions/v1beta1/namespaces/default/replicasets Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/apis/extensions/v1beta1/namespaces/default/deployments Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/api/v1/configmaps Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/api/v1/namespaces/default/events Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/api/v1/namespaces/default/configmaps Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/api/v1/namespaces/default/replicationcontrollers Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/api/v1/namespaces/default/pods Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/api/v1/namespaces/default/secrets Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/apis/extensions/v1beta1/ingresses Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/api/v1/namespaces/default/services Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/apis/extensions/v1beta1/namespaces/default/replicasets Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/apis/extensions/v1beta1/namespaces/default/deployments Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/api/v1/configmaps Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/api/v1/namespaces/default/events Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/api/v1/namespaces/default/configmaps Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/api/v1/namespaces/default/replicationcontrollers Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/api/v1/namespaces/default/pods Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/api/v1/namespaces/default/secrets Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/apis/extensions/v1beta1/ingresses Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/api/v1/namespaces/default/services Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/apis/extensions/v1beta1/namespaces/default/replicasets Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/apis/extensions/v1beta1/namespaces/default/deployments Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/api/v1/configmaps Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/api/v1/namespaces/default/events Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/api/v1/namespaces/default/configmaps Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/api/v1/namespaces/default/replicationcontrollers Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/api/v1/namespaces/default/pods Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/api/v1/namespaces/default/secrets Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/apis/extensions/v1beta1/ingresses Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/api/v1/namespaces/default/services Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/apis/extensions/v1beta1/namespaces/default/replicasets Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/apis/extensions/v1beta1/namespaces/default/deployments Failed to load resource: the server responded with a status of 404 ()
https://{IP}/k8s/api/v1/configmaps Failed to load resource: the server responded with a status of 404 ()
![screen shot 2016-10-14 at 11 13 51 am](https://cloud.githubusercontent.com/assets/21086784/19398056/9c50e090-91ff-11e6-878e-72ebe72cf32e.png)

**Comment by gregdaybreak** [comment]
In clicking around a bit It seems like it is trying to go to URL's like https://{IP}/k8s/api/v1/configmaps which does not seem to be valid. When I remove the k8s and just go to something like https://{IP}/api/v1/configmaps it returns data.

**Comment by christian-posta** [comment]
any thoughts @jstrachan or @rawlingsj ? I know you guys are working on some of the persistent/configmap stuff wrt to these deployments

**Comment by rawlingsj** [comment]
There were some nginx issues recently which have since been reverted from the image we use.  I also noticed I needed to create a gcloud firewall entry so that I could hit the nginx loadbalancer.  Not sure if this is a new thing as I've not needed to do that before.

You might want to restrict this to be for certain clusters etc but as an example I ran:
```
gcloud compute firewall-rules create fabric8 --allow tcp:80,icmp
```
created a new cluster https://console.cloud.google.com/kubernetes/

then ran 

```
gofabric8 deploy
```

any it worked, after a few mins the fabric8 console popped up.  I did have to go into incognito via chrome to avoid a caching issue with xip but the console was accessible.
